### PR TITLE
misc enhancements to the UI

### PR DESCRIPTION
**Pull request by tardyp**
given feedbacks from Jon__ on IRC, here are some easy enhancements to the UI

- use title, titleURL from the master config
- only show configured builders in builder page
- decent slaves page
- use of the settings framework

Signed-off-by: Pierre Tardy <pierre.tardy@intel.com>

### CODE REVIEW

File: www/base/src/app/about/about.controller.coffee [hunk 1]
```diff
@@ -1,15 +1,11 @@
 class About extends Controller
     constructor: ($scope, config, buildbotService) ->
-        # translate github urls to http urls for link generations
-        #for module in bower_configs
-        #    module.http_url = module.repository?.url?.replace("git://", "http://")
-        # todo should get that info from data api
+        # todo should get that info from data api: see http://trac.buildbot.net/ticket/3175
         $scope.config =
             bbversion: "0.9.0"
```
Comment (sa2ajj) [inline www/base/src/app/about/about.controller.coffee]: what about these two?

Comment (tardyp) [inline www/base/src/app/about/about.controller.coffee]: this is part of another known (linked) bug. will take a little bit more time

**Comment by tardyp** [comment]
dont merge yet. I missed some stuff

**Comment by tardyp** [comment]
I fixed some issues with connected_to.

But it looks that there are bugs in the core.
configured_buildslaves is not managed correctly so the displayed information is wrong. I'll look at it tomorrow.

**Comment by sa2ajj** [comment]
Travis said "it looks good" :)

**Comment by tardyp** [comment]
I think you can merge it.

**Comment by sa2ajj** [comment]
The last change causes two test cases to fail:

```
exceptions.SyntaxError: invalid syntax (sample.cfg, line 103)
buildbot.test.integration.test_configs.RealConfigs.test_sample_config
```

**Comment by tardyp** [comment]
oups fixed

**Pull request merged**